kernel/kernel-impl/src/main/webapp/WEB-INF/db-components.xml

```diff
@@ -491,25 +463,21 @@
 			</props>
 		</property>
 	</bean>
-	-->
 
 	<!--  The "Global" Session Factory -->
-	<!-- uncomment this bean for EntityManager
 	<bean id="org.sakaiproject.springframework.orm.hibernate.GlobalSessionFactory"
 		  class="org.springframework.orm.jpa.vendor.HibernateJpaSessionFactoryBean">
-		<property name="entityManagerFactory" ref="org.sakaiproject.springframework.orm.jpa.EntityManagerFactory"/>
+		<property name="entityManagerFactory" ref="entityManagerFactory"/>
 	</bean>
-	-->
 
 	<!--  The "Global" Transaction Manager -->
 	<bean id="org.sakaiproject.springframework.orm.hibernate.GlobalTransactionManager" class="org.springframework.orm.hibernate4.HibernateTransactionManager">
 		<property name="sessionFactory" ref="org.sakaiproject.springframework.orm.hibernate.GlobalSessionFactory"/>
 	</bean>
-	<!-- This jpa tx manager currently does not work as it behaves differently than hibernate's tx manager
-	<bean id="org.sakaiproject.springframework.orm.jpa.GlobalTransactionManager" class="org.springframework.orm.jpa.JpaTransactionManager">
-		<property name="entityManagerFactory" ref="org.sakaiproject.springframework.orm.jpa.EntityManagerFactory"/>
+	<!-- JPA tx manager, it behaves differently than hibernate's tx manager -->
+	<bean id="transactionManager" class="org.springframework.orm.jpa.JpaTransactionManager">
```

Comment:
Id renaming to fit with Spring Boot requirements:

org.springframework.boot.web.support.ErrorPageFilter.forwardToErrorPage Forwarding to error page from request [/rest/rubrics] due to exception [No bean named 'transactionManager' available: 
		No matching PlatformTransactionManager bean found for qualifier 'transactionManager' - neither qualifier match nor bean name match!]
org.springframework.beans.factory.NoSuchBeanDefinitionException: 
	No bean named 'transactionManager' available: No matching PlatformTransactionManager bean found for qualifier 'transactionManager' - neither qualifier match nor bean name match!